src/Solvers/plan_transforms.jl

```diff
@@ -47,6 +47,18 @@ function plan_backward_transform(A::CuArray, topo, dims, planner_flag)
     return CUDA.CUFFT.plan_ifft!(A, dims)
 end
 
+forward_orders(::Type{Periodic}, ::Type{Bounded} , ::Type{Bounded})  = (3,2,1)
+forward_orders(::Type{Periodic}, ::Type{Bounded} , ::Type{Periodic}) = (2,1,3)
+forward_orders(::Type{Bounded} , ::Type{Periodic}, ::Type{Bounded})  = (1,3,2)
+forward_orders(::Type{Bounded} , ::Type{Bounded} , ::Type{Periodic}) = (1,2,3)
+forward_orders(::Type{Bounded} , ::Type{Bounded} , ::Type{Bounded})  = (1,2,3)
+
+backward_orders(::Type{Periodic}, ::Type{Bounded} , ::Type{Bounded})  = (1,2,3)
+backward_orders(::Type{Periodic}, ::Type{Bounded} , ::Type{Periodic}) = (3,1,2)
+backward_orders(::Type{Bounded} , ::Type{Periodic}, ::Type{Bounded})  = (2,1,3)
+backward_orders(::Type{Bounded} , ::Type{Bounded} , ::Type{Periodic}) = (3,1,2)
+backward_orders(::Type{Bounded} , ::Type{Bounded} , ::Type{Bounded})  = (1,2,3)
```

Comment:
Not a big deal but I might just add spaces after the commas on the right side of the `=` sign.

PS: You probably noticed we're being a little naughty by dispatching on datatypes... Hoping we can resolve #1119 soon-ish since this has permeated the code pretty deeply.

Reply:
> PS: You probably noticed we're being a little naughty by dispatching on datatypes... Hoping we can resolve #1119 soon-ish since this has permeated the code pretty deeply.

I did notice that. I'd probably lean towards switching to type instances.

Reply:
Updated.